packages/wonder-blocks-clickable/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "@khanacademy/wonder-blocks-clickable",
+    "version": "1.0.0",
```

Comment:
I'd err on the side of staying `private: true` and version `0.1.0`. It's easier to go from this to public release than to discover a problem at public release and have to back pedal. I prefer to plan for the worst rather than hope for the best. What do you think?

Reply:
+1 for `private: true` and adding `0.1.0`. By adding this initial version, lerna will try to publish `1.0.0` instead of `1.0.1` (or something similar). Nothing that impacts at all, but good for keeping semver consistency.